tests/many_to_one/tests.py

```diff
@@ -522,6 +523,24 @@ def test_fk_assignment_and_related_object_cache(self):
         self.assertIsNot(c.parent, p)
         self.assertEqual(c.parent, p)
 
+    def test_save_nullable_parent_then_child(self):
+        author = Author()
+        book = Book(author=author)
+        author.save()
+        book.save()
+        book.refresh_from_db()
+        self.assertEqual(book.author, author)
+
+    def test_save_nullable_parent_then_child_to_field(self):
+        author_id = "jeff"
+        author = Author(name=author_id)
+        book = ToFieldBook(author=author)
+        author.save()
+        book.save()
+        book.refresh_from_db()
+        self.assertEqual(book.author, author)
+        self.assertEqual(book.author_id, author_id)
```

Comment:
I don't know when it has been fixed.

I don't believe "to_field" was within the scope of the ticket. I just thought it would be a good idea to add it in since it was brought up. I guess this test should've been run before the changes were made to ensure that it is valid. Since it's not within the scope of the ticket? Should I remove it?

Reply:
You can move it to a separate (first) commit in this PR.